app/views/password_resets/show.html.haml

```diff
@@ -0,0 +1,14 @@
+%section.reset_password.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      = form_tag password_resets_path do
+        %header
+          %h1 Reset Your Password
+        %fieldset.form-group
+          = label_tag :password, "New password"
+          .row
+            .col-sm-4
+              = text_field_tag :password
```

Comment:
This should definitely be `password_field_tag` otherwise anyone else behind user, who can see his screen can identify the password too.